chef/cookbooks/resolver/recipes/default.rb

```diff
@@ -29,6 +29,7 @@
 end
 
 dns_list << node[:dns][:nameservers]
+dns_list = dns_list.flatten.sort.uniq
```

Comment:
@vuntz I did not want to change line 26 as I want to be sure that the complete list is unified, not only a part of it.